### PR TITLE
Add some more ca to rotate

### PR DESCRIPTION
**Pull request by asalkeld**
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/specs/fakerp/carotation.go [hunk 1]
```diff
@@ -28,14 +28,18 @@ var _ = Describe("Ca Rotation E2E tests [CaRotation][Fake][LongRunning]", func()
 		Expect(after).NotTo(BeNil())
 
 		By("Verifying that the correct ca certificates have been updated")
+		Expect(before.Config.Certificates.Ca.Cert).To(Equal(after.Config.Certificates.Ca.Cert))
```
Comment (ehashman) [inline test/e2e/specs/fakerp/carotation.go]: Do we also want to rotate this CA, or is this a CA we can't rotate?

**Comment by ehashman** [comment]
/retest

Looks like a timeout flake.

**Comment by ehashman** [comment]
/retest

flake #1438

File: test/e2e/specs/fakerp/carotation.go (hunk 1 above)
Comment (asalkeld) [reply test/e2e/specs/fakerp/carotation.go]: doing them bit by bit. This is a harder one as it's used all over the place.

**Pull request merged**